cdap-docs/admin-manual/source/operations/preferences.rst

```diff
@@ -119,6 +119,14 @@ the argument will be set for all programs that use ``myTable``. In order to set
 program ``myMR``, the dataset argument can be nested inside the program scope as
 ``mapreduce.myMR.dataset.myTable.cache.seconds=30``.
 
+Overriding Logical Start Time with Runtime Arguments
+====================================================
+
+The runtime argument ``logicalStartTime`` can be used to override the logical start time of a Workflow, MapReduce,
+or Spark program that has been manually started.  Instead of using the current time as the logical start time,
```

Comment:
nit - you have 2 spaces after some periods, but only 1 space after another period.